single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -1321,9 +1321,9 @@ private void drawNodeInfos(ElectricalNodeInfo nodeInfo, double xShift, double yS
         circle.setAttribute("cy", String.valueOf(yShift));
         circle.setAttribute("r", String.valueOf(CIRCLE_RADIUS_NODE_INFOS_SIZE));
         circle.setAttribute("stroke-width", String.valueOf(CIRCLE_RADIUS_NODE_INFOS_SIZE));
-        if (!styles.isEmpty()) {
-            circle.setAttribute(CLASS, String.join(" ", styles));
-        }
+        List<String> classStyles = new ArrayList<>(styles);
+        classStyles.add(NODE_INFOS);
```

Comment:
I think that this `NODE_INFOS` style should be directly added in the `AbstractBaseVoltageDiagramStyleProviders::getBusStyles` & `DefaultDiagramStyleProvider::getBusStyles`